packages/venia-ui/lib/drivers/adapter.js

```diff
@@ -21,11 +17,8 @@ import { shrinkGETQuery } from '../util/shrinkGETQuery';
  * The tradeoff is that we may be creating an instance we don't end up needing.
  */
 const preInstantiatedCache = new InMemoryCache({
-    dataIdFromObject: cacheKeyFromType,
-    fragmentMatcher: new IntrospectionFragmentMatcher({
-        // UNION_AND_INTERFACE_TYPES is injected into the bundle by webpack at build time.
-        introspectionQueryResultData: UNION_AND_INTERFACE_TYPES
-    })
+    typePolicies,
+    possibleTypes: POSSIBLE_TYPES
```

Comment:
May want to drop a comment where `POSSIBLE_TYPES` comes from since it's not otherwise present in this file

Reply:
Done 8eafc91d36788000ed3e2e6bf137c670a5b71b46